docs/_documentations/remotedeploy-vscode.md

```diff
@@ -64,14 +66,14 @@ This example uses Docker Hub:
 
 Enter:
 
-- Your Docker Hub user name
+- Your Docker Hub username
 - Your Docker Hub password
-- Your namespace, which is usually the same as your user name
+- Your namespace, which is usually the same as your username
 
```

Comment:
Docker Hub or container registry?

Reply:
Substituted the more generic term "container registry" for Docker Hub.